libs/barista-components/experimental/datepicker/src/timeinput.ts

```diff
@@ -128,14 +142,17 @@ export class DtTimeInput {
   /**
    * @internal
    * Add the focus switch from the hour input to the minute input when the user typed in 2 digits.
+   * Note: the nativeElement values are used here because otherwise the number values with leading 0s
+   * would be converted to 1 digit (e.g. 05 would be 5), which would falsely evaluate as 1 digit numbers
+   * and not switch the focus.
```

Comment:
Sorry, I needed to read this 10 times to understand that this has nothing to do with leading 0s (zero seconds) and rather to do with leading zeros. 

```suggestion
   * Note: the nativeElement values are used here because otherwise the number values with leading zeros
   * would be converted to 1 digit (e.g. 05 would be 5), which would falsely evaluate as 1 digit numbers
   * and not switch the focus.
```

Reply:
Oh, yeah, didn't think about the other meaning, sorry about that :))